### DISCUSSION TITLE
How to connect SeleniumBase to already opened window of custom driver(using Driver()) it connects via Selenium

### DISCUSSION THREAD

**Discussion by STARKILLER-1** [seleniumbase/SeleniumBase#2729] [Q&A] [answered]
I am trying to connect a Dolphin {anty} chromedriver to Seleniumbase? 
I have a port to connect from dolphin api
Selenium works fine Here are some docks:
https://dolphin-anty.com/blog/api-avtomatizaciya-brauzera-chto-takoe-selenium-puppeteer-i-playwright/
DOWNLOAD LINK of custom driver here(selenium part of article hyperlink): https://intercom.help/dolphinteam/ru/articles/7127390-базовая-автоматизация-dolphin-anty

**Comment by mdmintz** [comment, marked as answer]
Pass the `--remote-debugging-port` via `chromium_arg`. Eg:

```python
with SB(chromium_arg="--remote-debugging-port=7890") as sb:
    # ...
```

If that doesn't work, then the browser probably isn't compatible.

**Reply by STARKILLER-1** [reply]
It works! but SB opens it's own empty driver window with data:, in address line. but when I type headless2=True all is good! Feels like magic! btw all actions work(clicks etc.)

**Reply by STARKILLER-1** [reply]
Thank you!

**Comment by eshakazmi8082** [comment]
Hey @STARKILLER-1 Can you share the code to connect seleniumbase with Dolphin {anty} browser.? i am also working on a similar project.
